nucypher/control/interfaces.py

```diff
@@ -39,12 +40,19 @@ def __init__(self, implementer=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     @classmethod
-    def connect_cli(cls, action):
+    def connect_cli(cls, action, exclude: Optional[Set[str]] = None):
```

Comment:
Where is this kwarg being called?  The idea here is to provide an alternative method for CLI option exclusion?

Reply:
The general idea is that the schema can still be used for populating CLI options, but allow for some CLI options to be modified. 

More specifically, it is used here - https://github.com/derekpierre/nucypher/blob/porter-retrieval/nucypher/cli/commands/bob.py#L347 with the `nucypher bob retrieve-and-decrypt` CLI command. In this case, the `retrieve-and-decrypt` command from the CLI can be used without requiring an `alice_verifying_key` to be specified via `--alice-verifying-key` option which is required by the schema. An Alice card can be specified from the CLI via `--alice` and the verifying key obtained from the Card API. 

In this case `--alice-verifying-key` needs to be optional and not required. Either Alice's verifying key is provided directly via `--alice-verifying-key` or indirectly via `--alice`.